### PR TITLE
ref 3420 update private room screen

### PR DESCRIPTION
**Pull request by tuanda5353**
### Summary:
- [Feature] Chat screen
### Link Issue:
- Update private room screen

### CODE REVIEW

**Comment by tuanda5353** [comment]
ready

File: app/src/main/java/framgia/com/ichat/screen/privateroom/PrivateRoomPresenter.java [hunk 1]
```diff
@@ -46,15 +46,17 @@ public void onFailure(@NonNull Exception e) {
     }
 
     @Override
-    public void getPrivateRooms() {
+    public void getPrivateRooms(final String id) {
         mRepository.getPrivateRooms(new ValueEventListener() {
             @Override
             public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                 List<Room> listRoom = new ArrayList<>();
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/framgia/com/ichat/screen/privateroom/PrivateRoomPresenter.java]: listRoom rename to rooms
also PrivateRoomFragment Class

File: app/src/main/java/framgia/com/ichat/screen/privateroom/PrivateRoomFragment.java [hunk 2]
```diff
@@ -48,9 +49,9 @@ protected void initComponents() {
     @Override
     protected void initData(Bundle savedInstanceState) {
         mPresenter = new PrivateRoomPresenter(PrivateRoomRepository.getInstance
-                (PrivateRoomRemoteDatasource.getInstance(FirebaseDatabase.getInstance())));
+                (PrivateRoomRemoteDatasource.getInstance(FirebaseDatabase.getInstance(), FirebaseAuth.getInstance())));
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/framgia/com/ichat/screen/privateroom/PrivateRoomFragment.java:52]: dòng 24 vượt 80 kí tự

**Comment by tuanda5353** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**